tests/integration/integration-test.sh

```diff
@@ -16,6 +16,13 @@ RETRY=3
 ## Backup helm/values.yaml
 cp helm/values.yaml values.yaml.backup
 
+## Join ghcr integration yaml
+if ! [[ -z "${IMAGE+x}" || -z "${imagepullsecret+x}" ]]; then
```

Comment:
if not (x unset or y unset) is way harder on cognition than if x and y

Reply:
sorry, I don't think I fully understand the proposal

Reply:
please see my fix